ideainspect.groovy

```diff
@@ -85,13 +74,20 @@ if (!resultPath.mkdirs()) {
 }
 
 //
+// --- Running VMOptions Modifier
+//
+if(cliOpts.sc){
+  modifyIdeaProps(cliOpts)
+}
+
+//
 // --- Actually running IDEA
 //
 
 //  ~/projects/dashboard.git/. ~/projects/dashboard.git/.idea/inspectionProfiles/bens_idea15_2015_11.xml /tmp/ -d server
 def ideaArgs = [ideaPath.path, "inspect", rootDir.absolutePath, profilePath.absolutePath, resultPath.absolutePath, "-v1"]
 if (cliOpts.d) {
-  ideaArgs << "-d" << cliOpts.d
+  ideaArgs << "-d" << rootDir.absolutePath + "/" + cliOpts.d
```

Comment:
Woops, I didn't mean for this one to go through, this is just something that we used, because our absolute path changes and we wanted to test local. I'll take it out.